repos/c2sm/packages/icon/package.py

```diff
@@ -547,6 +556,10 @@ def configure_args(self):
             # flags (i.e. not the linker -l<library> flags), therefore we put
             # the value to the config_flags directly.
             config_vars['LIBS'].extend(cuda_host_compiler_stdcxx_libs)
+        else:
+            raise error.UnsupportedPlatformError(
+                'only gpu=openacc with +cuda supported with spack for gpu build'
+            )
```

Comment:
```suggestion
```
This code is unreachable, because the other branches already catch all possible cases.